### PR TITLE
chore: add call formatter

### PR DESCRIPTION
**Pull request by jonybur**
# Description

## Problem\*

Resolves 

## Documentation

- [ ] This PR requires documentation updates when merged.

  

  - [ ] I will submit a noir-lang/docs PR.

  

  - [ ] I will request for and support Dev Rel's help in documenting this PR.

  
  

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

**Comment by jfecher** [comment]
Can you run `cargo fmt`?

File: tooling/nargo_fmt/src/visitor/expr.rs [hunk 1]
```diff
@@ -39,6 +39,16 @@ impl FmtVisitor<'_> {
                     self.format_expr(infix.rhs)
                 )
             }
+            ExpressionKind::Call(call_expr) => {
+                let formatted_func = self.format_expr(*call_expr.func);
+                let formatted_args = call_expr
+                    .arguments
+                    .iter()
```
Comment (jfecher) [inline tooling/nargo_fmt/src/visitor/expr.rs:46]: Thank you, cargo fmt. `.arguments` is completely worthy of being on its own separate line.

**Review by jfecher — APPROVED** [review]
LGTM

File: tooling/nargo_fmt/src/visitor/expr.rs [hunk 2]
```diff
@@ -39,6 +39,15 @@ impl FmtVisitor<'_> {
...
+                    .map(|arg| self.format_expr(arg.clone()))
```
Comment (TomAFrench) [inline tooling/nargo_fmt/src/visitor/expr.rs:47]:
```suggestion
                    .into_iter()
                    .map(|arg| self.format_expr(arg))
```

**Review by TomAFrench — DISMISSED** [review]

**Comment by TomAFrench** [comment]
Oof, what did I do?

**Review by TomAFrench — APPROVED** [review]

**Review by kevaundray — APPROVED** [review]

**Pull request merged**